src/main/java/org/csanchez/jenkins/plugins/kubernetes/volumes/PodVolume.java

```diff
@@ -41,8 +44,26 @@
     // Where to mount this volume in the pod.
     public abstract String getMountPath();
 
+    // Builds a Volume model with the given name.require podName to generate pvc name
+    public Volume buildVolume(String volumeName, String podName){
+        return buildVolume(volumeName);
+    };
+
     // Builds a Volume model with the given name.
-    public abstract Volume buildVolume(String volumeName);
+    @Deprecated
```

Comment:
Some javadoc here would be good IMO. it's unclear why this is throwing `UnsupportedOperationException` and why subclases are granted to have an override of this, unless you come to this PR and see that the method was `abstract` before this change.